python/nav/web/templates/seeddb/edit_prefix.html

```diff
@@ -2,5 +2,9 @@
 {% load crispy_forms_tags %}
 {% block crispyfields %}
 {{ block.super }}
-{{ vlan_form|crispy }}
+{% if vlan_form.attrs %}
+    {% include 'custom_crispy_templates/_form_content.html' with form=vlan_form %}
+{% else %}
+    {{ vlan_form }}
+{% endif %}
```

Comment:
WIll the `{% block crispyfields %}` and `{% load crispy_forms_tags %}` be fixed in a different PR? Some weird stuff happening with this form

Reply:
Good catch, no, I need to remove `{% load crispy_forms_tags %}`! 

But the `{% block crispyfields %}` is necessary, since that is the name that it also has in `python/web/templates/seeddb/edit.html`. 
But when we have removed crispy also from there (in #3128) then we can rename that block to something more sensible